optaplanner-docs/src/main/asciidoc/DesignPatterns/DesignPatterns-chapter.adoc

```diff
@@ -9,92 +9,74 @@
 
 
 [[designPatternsIntroduction]]
-== Design Patterns Introduction
-
-These design patterns list and solve common design challenges.
+== Design patterns introduction
 
+The design patterns in this section list and solve common design challenges.
+//Need to explain what design patterns are.
 
 [[domainModelingGuide]]
-== Domain Modeling Guide
-
-A good model can greatly improve the success of your planning implementation.
-These guidelines help to design a good model:
-
-. *Draw a class diagram* of your domain model. _Normalize_ it get rid of duplicate data.
-** Write down some sample instances for each class.
-*** For example in employee rostering, the samples for the `Employee` class are _Ann_, _Bert_ and _Carl_.
-
-. *Determine which relationships (or fields) change during planning*. Color them orange.
-Often there is only one such relationship (or field).
-One side of these relationships will become a planning variable later on.
-** For example in employee rostering, the `Shift` to `Employee` relationship changes during planning,
-so it is orange.
-Other relationships, such as from `Employee` to `Skill`, are immutable during planning
-because Planner can't decide to give an employee an extra skill.
-
-. If there are multiple relationships (or fields): *check for <<shadowVariable,shadow variables>>*.
-A shadow variable does change during planning,
-but its value can be calculated based on one or more genuine planning variables, without dispute.
-Color those shadow relationships (or fields) purple.
-** Only one side of a bi-directional relationship can be a genuine planning variable,
-the other side will become an <<bidirectionalVariable,inverse relation shadow variable>> later on.
-Keep those relationships in orange.
+== Domain modeling guidelines
```

Comment:
Originally, we used scientific capitalization of titles, so like this.
But on request of product-docs, we started following the guideline of english article captilization of titles (so every new word has a title).

Which is it? :)
I am happy to switch back to scientific style, but no in the PR, but in PR that does that (and only that) for the entire codebase.